### ISSUE TITLE
Open data per default: building CeTrAn 5.0 (R code)

### ISSUE THREAD

**Issue by acabunoc** [mozillascience/global-sprint-2016#42]
**[ Project Lead ]** @jcolomb
**[ GitHub Repo ]** https://github.com/jcolomb/CeTrAn
**[ Track ]** Tools: you are building a tool others can help you build, Open Data: you have data others can use and play with during the sprint
**[ Level ]** Intermediate

### Description
CeTrAn is a R suite of code to analyse animal behavior data as recorded via the trajectory of their centre of gravity (centroid trajectory analysis). It was used to analyse data obtained in the Buridan's paradigm in fruit fly, but could be extended.
The main goals of the sprint would be:

- Set a metadata standard which would also accept rodent open field data
- Expand the code to publish the data and metadata on Figshare before the analysis starts (using the Rfigshare package)
- Build a shiny module to replace (and expand) the GUI.
- Present the results in a authorea or overleaf document, make it a pre-print article and (maybe later) send it for peer review at plos biology (?).

I will procure some data to test the system.

Associated goals could be:
- Clean the code and add a better documentation
- Create a R package
- Analyse some data from mice open field experiments

***
## Want to Contribute?
Join us at the [Global Sprint June 2-3](https://www.mozillascience.org/global-sprint-2016). Leave a comment in this issue to let the project lead know you're interested in contributing during #mozsprint 2016!
***

## Note to the Project Lead
Congrats, @jcolomb! This is your official project listing for the [Mozilla Science Global Sprint 2016](https://www.mozillascience.org/global-sprint-2016). To confirm your submission, please complete the following:

- [x] Provide a GitHub repository for work and discussion on your project
- [ ] Confirm in a comment that at least one person will be available to review and answer questions on this project from 9-5 in their timezone on both June 2 & 3.

#### Do you want to become a featured project? :tada:
Here are some exercises that will help your project be more inviting to new contributors. We hope you'll try to complete some of these as you prepare for #mozsprint.

If you complete *all* the exercises, your project will be eligible to be featured in our [collection of open source science projects](https://mozillascience.org/collaborate)! Once you've finished this list, contact @acabunoc to submit your project for review.

* [ ] [README.md](http://mozillascience.github.io/working-open-workshop/writing_readme/):
* [ ] [LICENSE](http://choosealicense.com/):
* [ ] [CONTRIBUTING.md](http://mozillascience.github.io/working-open-workshop/contributing/):
* [ ] [Roadmap or an Issue Tracker (with tasks to complete)](http://mozillascience.github.io/working-open-workshop/roadmapping/):
* [ ] [Data Reuse Plan](http://mozillascience.github.io/working-open-workshop/data_reuse/) (if applicable):
* [ ] [Code of Conduct](http://mozillascience.github.io/working-open-workshop/code_of_conduct/):
* [ ] [Personas](http://mozillascience.github.io/working-open-workshop/personas_pathways/):

**Comment by offray** [comment]
Hi,

It seems that I'm susbcribed to a lot of other project notifications 
like this one, How can I unsubscribe from all?

Cheers,

Offray

**Comment by acabunoc** [comment]
Hey @offray -  update your notifications at the top of the page:
![screen shot 2016-05-19 at 08 18 30](https://cloud.githubusercontent.com/assets/617994/15398897/52e0a38e-1d9a-11e6-8307-e2bb112e141d.png)

**Comment by offray** [comment]
Thanks Abigail. I did it for my project, but I was also in the 
notifications of the global sprint.

Cheers,

Offray

**Comment by acabunoc** [comment]
Yes @offray, make sure you do this at the **top of the page** to unwatch the entire repo. You're still listed as watching.

You can also go here to change your github settings: https://github.com/settings/notifications

**Comment by offray** [comment]
I think now I have done it right.

Thanks,

Offray

**Comment by jcolomb** [comment]
timezone is GMT+2 Berlin time.
